src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/BookingFacade.kt

```diff
@@ -44,10 +45,11 @@ class BookingFacade(
   private val emailService: EmailService,
   private val notificationRepository: NotificationRepository,
   private val outboundEventsService: OutboundEventsService,
-  private val locationsInsidePrisonClient: LocationsInsidePrisonClient,
+  private val locationsService: LocationsService,
```

Comment:
Now using location service here as this provides what we need for extra room attributes to pass onto email.s